plugin/pkg/admission/gc/gc_admission_test.go

```diff
@@ -68,6 +68,15 @@ func (fakeAuthorizer) Authorize(a authorizer.Attributes) (authorizer.Decision, s
 		return authorizer.DecisionAllow, "", nil
 	}
 
+	if username == "non-node-deleter" {
+		if a.GetVerb() == "delete" && a.GetResource() == "nodes" {
+			return authorizer.DecisionNoOpinion, "", nil
```

Comment:
I assume there's not other authorizers running and therefore this is the same as a reject? Would it make the test more obvious if we changed these to explicit rejections?